src/v_trans.c

```diff
@@ -204,7 +204,7 @@ byte V_Colorize (byte *playpal, int cr, byte source)
         }
         else if (cr == CR_BLUE || cr == CR_BLUE2)
         {
-            hsv.x = (float) 240./360.;
+            hsv.x = (float) (240./360.);
```

Comment:
Alternative fix: `hsv.x = 240.f/360.f;`
Why not use `double` in v_trans.c? By default, all calculations are performed in double precision.

Reply:
Hm, not sure, I have taken this code from somebody else and would just leave it untouched as it "just worked". 😁